core/src/main/java/org/sonarsource/sonarlint/core/container/storage/ProjectStoragePaths.java

```diff
@@ -31,7 +31,7 @@
   /**
    * Version of the storage. This should be incremented each time an incompatible change is made to the storage.
    */
-  public static final String STORAGE_VERSION = "3";
+  public static final String STORAGE_VERSION = "3 ";
```

Comment:
Is this extra space intended?

Reply:
Of course no, I don't know how it ended up here :disappointed: